### DISCUSSION TITLE
Could you add some  different style entry and exits?

### DISCUSSION THREAD

**Discussion by Flammable-Nate** [BEEmod/BEE2-items#4064] [Tech Support / Q&A] [answered]
What I mean is if you want to you could add say for example a portal 1 exit gate to an original clean theme and a 80's theme exit for the same level if you could do that it would be awesome!

**Comment by Flammable-Nate** [comment]
Actually your other plans sound better than what I'm asking for

**Reply by Flammable-Nate** [reply]
#3891

**Comment by slambo311** [comment, marked as answer]
Check the FAQ, they're not doing style mixing.
